src/Service.Host/client/src/components/parts/tabs/GeneralTab.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
```

Comment:
Are there any tests for these components btw?

Reply:
none at all.. They're pretty massive components and I can't really think of any useful tests. They're pretty much exclusively made up of shared components though which should be tested (in theory) so it's not entirely untested code.